clients/src/main/java/org/apache/kafka/common/protocol/Readable.java

```diff
@@ -18,12 +18,14 @@
 package org.apache.kafka.common.protocol;
 
 import java.nio.charset.StandardCharsets;
+import java.util.UUID;
 
 public interface Readable {
     byte readByte();
     short readShort();
     int readInt();
     long readLong();
+    UUID readUUID();
```

Comment:
Please add a default implementation of this to here.  The subclasses shouldn't need it.